Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -1994,6 +1994,13 @@ theorem prod_int_mod (s : Finset α) (n : ℤ) (f : α → ℤ) :
   (Multiset.prod_int_mod _ _).trans <| by rw [Finset.prod, Multiset.map_map]; rfl
 #align finset.prod_int_mod Finset.prod_int_mod
 
+@[to_additive (attr := simp)]
+theorem smul_prod
+    [CommMonoid α] [Monoid β] [MulAction β α] [IsScalarTower β α α] [SMulCommClass β α α]
+    (s : Finset α) (b : β) :
+    b ^ card s • ∏ x in s, x = ∏ x in s, b • x := by
```

Comment:
I'm not sure that RHS is simpler than LHS. In many cases, I would move in the inverse direction.

Reply:
This order is to match an existing order elsewhere; so I'd be inclined to just remove `simp`. Agreed about `f` and the other variant.